src/EditorFeatures/CSharpTest/EditAndContinue/ActiveStatementTests.cs

```diff
@@ -622,6 +622,28 @@ static void Foo(int a)
             edits.VerifyRudeDiagnostics(active);
         }
 
+        [WpfFact]
+        public void Delete_Entire_Namespace()
+        {
+            string src1 = @"
+namespace N {
```

Comment:
Brace on the next line please (unless you're testing something to do with its position)